cypress/support/pages/admin/pageAdministration.js

```diff
@@ -505,6 +509,51 @@ class Management {
 		cy.get(Management.#externalToolsTable).should("be.visible");
 	}
 
+	seeEmptyExternalToolTable() {
+		cy.get(Management.#externalToolsTable).should("not.exist");
+	}
+
+	seeExternalToolConfigurationPage() {
+		cy.url().should("include", "/administration/school-settings/tool-configuration");
+	}
+
+	seeToolConfigurationInfoText() {
+		cy.get(Management.#externalToolConfigInfotext).should("be.visible");
+	}
+
+	seeExternalToolConfiguratorPageTitle() {
+		cy.get(Management.#externalToolConfigPageTitle).should("exist");
+	}
+
+	seeSelectedExternalTool(toolName) {
+		cy.get(Management.#toolConfigurationSelectedItem).should("contain.text", toolName);;
+	}
+
+	seeCustomParameterFormContains(paramName, value) {
+		cy.get(`[data-testid="${paramName}"]`).find("input").should("have.value", value);
+	}
+
+	fillInCustomParameter(paramName, value) {
+		cy.get(`[data-testid="${paramName}"]`).find("input").clear().type(value);
+	}
+
+	externalToolIsNotVisibleInToolSelection(toolName) {
+		cy.get(Management.#toolSelection).click();
+		cy.get(Management.#toolConfigurationSelectItem).contains(toolName).should("not.exist");
+	}
+
+	seeToolErrorAlert() {
+		cy.get(Management.#toolErrorAlert).should("be.visible");
+	}
+
+	seeDeactivatedCheckBox(value) {
```

Comment:
is the value important? If yes, consider maybe to destructure the method to "isNotChecked" or "isChecked", if not it is ok

Reply:
fixed